### ISSUE TITLE
Logout functionality

### ISSUE THREAD

**Issue by dhont** [code4romania/monitorizare-vot#241]

**Comment by jrstrayhorn** [comment]
@aniri - I noticed that you can't log out of the admin app right now... thinking about working on this one.. would you need an API endpoint to do logout from Angular frontend?  

Would you just need to do all Angular frontend changes like so?

- add logout button/link in top nav (just functionality first 😊)
- set the token value to undefined via tokenService (which would remove token from local storage)
- redirect back to login page

**Comment by aniri** [comment]
@jrstrayhorn yep, we do need the logout functionality. For now, I think the second option you mentioned would be simpler :) 

Maybe even have the logout link shown like in the design https://www.figma.com/file/61NHR7fwmV1Dsv4Pb6F8T7/code4ro_mv_user-management_2018.04.21-(Copy)?node-id=1120%3A31
We could show the username in the right of the menu and have the logout option available when you click on the username. Or at least just the logout link :)

**Comment by jrstrayhorn** [comment]
Ok sounds good.. I will work on this!  

Should this item get moved to the monitorizare-vot-ong repo since all changes will be on the Angular frontend app?
